applications/StructuralMechanicsApplication/custom_elements/spring_damper_element.hpp

```diff
@@ -6,7 +6,7 @@
 //  License:         BSD License
 //                   license: StructuralMechanicsApplication/license.txt
 //
-//  Main authors:    Quirin Aumann
+//  Main authors:    Quirin Aumann, Aron Noordam
```

Comment:
```suggestion
//  Main authors:    Quirin Aumann
                                  Aron Noordam
```

usually authors are on separate lines